protocol/src/main/scala/io/mediachain/protocol/CborSerialization.scala

```diff
@@ -693,8 +693,8 @@ object CborSerialization {
 
     override def fromCValue(cValue: CValue): Xor[DeserializationError, MultihashReference] = {
       cValue match {
-        case CTaggedValue(tag, CBytes(value)) if tag == MultihashReference.cborLinkTag =>
-          MultiHash.fromBytes(value)
+        case CTaggedValue(tag, CString(value)) if tag == MultihashReference.cborLinkTag =>
+          MultiHash.fromBase58(value)
```

Comment:
Ok that makes sense :)

I'm kinda inclined to just make a multiaddress branch w/o dynamo support. We could also potentially try to specify dynamo addresses in another way, like using the REST API?